### PR TITLE
RATIS-1715. Support linearizable read in AsyncApi.

### PR DESCRIPTION
**Pull request by szetszwo**
See https://issues.apache.org/jira/browse/RATIS-1715

### CODE REVIEW

**Review by SzyWilliam — APPROVED** [review]
LGTM.

File: ratis-client/src/main/java/org/apache/ratis/client/api/AsyncApi.java [hunk 1]
```diff
@@ -40,13 +40,19 @@ public interface AsyncApi {
    */
   CompletableFuture<RaftClientReply> send(Message message);
 
+  /** The same as sendReadOnly(message, null, true). */
```
[the remark this answers is not in the document]
Comment (szetszwo) [reply ratis-client/src/main/java/org/apache/ratis/client/api/AsyncApi.java:43]: Good catch!  Thanks.

**Review by lokeshj1703 — APPROVED** [review]
@szetszwo Thanks for working on this! The changes look good to me. +1.

**Comment by szetszwo** [comment]
@lokeshj1703 , thanks a lot for reviewing this!

**Pull request merged**